portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -234,7 +235,7 @@ const postReportConfirmAndSend = async (req, res) => {
         ...req.session.utilisationReport,
         paymentOfficerEmail,
       };
-      return res.redirect('/utilisation-report-upload/confirmation');
+      return res.redirect(`${LANDING_PAGES.UTILISATION_REPORT_UPLOAD}/confirmation`);
```

Comment:
I'm not sure if it's correct to use `LANDING_PAGES` in this context, as the redirect isn't related to going to a landing page, it just happens that the `LANDING_PAGES` constant has what we want.

Maybe instead of `LANDING_PAGES` we make this more general, like
```
const UTILISATION_REPORT_UPLOAD = '/utilisation-report-upload';
const UTILISATION_REPORT_UPLOAD_CONFIRM_AND_SEND = `${UTILISATION_REPORT_UPLOAD}/confirm-and-send`;
const UTILISATION_REPORT_UPLOAD_CONFIRMATION = `${UTILISATION_REPORT_UPLOAD}/confirmation`;

export const ROUTES = {
  DEFAULT_LANDING: '/dashboard/deals/0', <-- maybe construct this from other routes too?
  UTILISATION_REPORT_UPLOAD,
  UTILISATION_REPORT_UPLOAD_CONFIRM_AND_SEND,
  UTILISATION_REPORT_UPLOAD_CONFIRMATION
} as const;
```
What do you think?

Reply:
My problem with that is that introducing this constant implies that every route should have a constant, which I don't think it should. To be totally honest I don't agree with the presence of this constant at all as no other route uses a constant, and I only looked at introducing this in the other places to keep the url linked. I would rather remove all the references to `LANDING_PAGES` in locations like this and only keep it in the places where you are routed to a "landing page" (login and the home route)

Reply:
Actually, I do think that `LANDING_PAGES` is a constant that should only be referenced in those locations, and these other files should use raw strings

Reply:
Yep, that works.